src/components/table/template.njk

```diff
@@ -0,0 +1,27 @@
+<table class="govuk-c-table {{ classes }}">
+  {% if caption %}
+  <caption class="govuk-c-table__caption {{captionSize}}">{{ caption }}</caption>
```

Comment:
yeah can do if captionSize then add whole modifier class. we will have some global heading styles like we have now e.g heading-medium, heading small so we could either apply that class (and its modifiers) or a new modifier just for this caption